style/CONTRIBUTE.md

```diff
@@ -124,3 +124,27 @@ restart the web server.
 
 Please prettify all files prior to submission. Run `make code_format` to format all
 code files with js-prettifier.
+
+## Highway Shield Contributor's Guide
+
+Highway shields are a key feature of the OpenStreetMap Americana style. This guide describes some of the style principles that contributors of highway shield artwork should consider when submitting new shields. The required elements are as follows:
+
+1. Shields are in .svg format, saved in "Optimized SVG" format in Inkscape
+2. Shields are 20px on the smallest dimension (this will be rasterized to 20px or 40x depending on display resolution)
+3. Shields are license-compatible (public domain or CC0)
+
+In addition, the following style guidelines are recommended:
+
+1. Use a 1.25px stroke size for line elements.
+2. Complex or intricate graphic elements should be simplified
+3. Background shading should be removed
+4. Route numbers should be large enough to be legible
```

Comment:
Another point to consider is that shields should consistently use squared or rounded corners, but not a mix of both.